translations/strings.yaml

```diff
@@ -271,6 +271,9 @@ notifications:
   VPNConnectedMessage:
     value: "Connected to %1"
     comment: "%1 is the name of the city the VPN is connected to."
+  VPNMultihopConnectedMessage:
+    value: "Connected to %1, through %2"
+    comment: "%1 is the name of the entry server, %2 is the name of the exit server."
```

Comment:
On second thoughts, is this comment correct? 

I think the variables are swapped in the comment, "through %2" means that's the hop server, doesn't it?

Reply:
Yes, nice catch. Updated the comment, thanks